app/policies/follow_requests_policy.rb

```diff
@@ -0,0 +1,20 @@
+class FollowRequestsPolicy < ApplicationPolicy
+  attr_reader :user, :follow_requests
+
+  def initialize(user,follow_requests)
+    @user = user
+    @follow_requests = follow_requests
+  end 
+
+  def create?
+    true
+  end
+
+  def destroy?
+    user == follow_requests.recipient || user == follow_requests.sender
+  end
+
+  def update?
+    user == follow_requests.recipient || user == follow_requests.sender
```

Comment:
you can call `destroy?` here if it's following the same authorization rules

Reply:
resolved